src/main/resources/assets/minecolonies/lang/en_us.json

```
@@ -1528,19 +1538,40 @@
   "com.minecolonies.coremod.gui.chat.cancel": "Try cancel that request!",
   "com.minecolonies.coremod.gui.chat.fulfill": "More details please!",
   "com.minecolonies.coremod.gui.research.title": "Research Branches",
-  "com.minecolonies.coremod.research.requirement.building.level": "Requires %s at least level %d!",
+  "com.minecolonies.coremod.gui.research.tier.header": "Level %d (%d hr)",
+  "com.minecolonies.coremod.research.research.toomanyinprogress.1": "Too many researches",
+  "com.minecolonies.coremod.research.research.toomanyinprogress.2": "already in progress",
+  "com.minecolonies.coremod.research.research.notenoughresources": "Missing Resources",
+  "com.minecolonies.coremod.research.research.missingrequirements": "Missing Requirements",
+  "com.minecolonies.coremod.research.research.buildingleveltoolow": "Too low University level",
+  "com.minecolonies.coremod.research.requirement.building.level": "Requires %s(s) totaling at least %d levels!",
+  "com.minecolonies.coremod.research.requirement.building.or": "or ",
+  "com.minecolonies.coremod.research.requirement.research": "Requires %s!",
+  "com.minecolonies.coremod.research.requirement.university.level": "Requires University Level %d",
+  "com.minecolonies.coremod.research.limit.onemaxperbranch": "Only one max-level Research can be unlocked per Branch",
```

```suggestion
  "com.minecolonies.coremod.research.limit.onemaxperbranch": "Only one max-level research can be unlocked per branch.",
```
If part of it is changed to lowercase, it all needs to be.